src/sentry/replays/tasks.py

```diff
@@ -24,11 +25,23 @@
 def delete_recording_segments(project_id: int, replay_id: str, **kwargs: Any) -> None:
     """Asynchronously delete a replay."""
     metrics.incr("replays.delete_recording_segments", amount=1, tags={"status": "started"})
-    archive_replay(project_id, replay_id)
+    publisher = initialize_replays_publisher(is_async=False)
+    archive_replay(publisher, project_id, replay_id)
     delete_replay_recording(project_id, replay_id)
     metrics.incr("replays.delete_recording_segments", amount=1, tags={"status": "finished"})
 
 
+@instrumented_task(
+    name="sentry.replays.tasks.delete_replay_recording_async",
```

Comment:
```suggestion
    name="sentry.replays.tasks.delete_replay_recording",
```
Since all celery tasks are async

Reply:
It's to distinguish it from the synchronous `delete_replay_recording` function.